Wikipedia/Code/TalkPageDataController.swift

```diff
@@ -117,9 +111,18 @@ class TalkPageDataController {
         }
     }
 
+    private func cachedFileName() -> String {
+        guard let languageCode = siteURL.wmf_contentLanguageCode else {
+            return "en"
```

Comment:
I'm not sure if defaulting to "en" is the best approach, but I am happy to discuss better options